### DISCUSSION TITLE
How Do I disable JQuery 1.12.x

### DISCUSSION THREAD

**Discussion by yyyuuu123** [joomla/joomla-cms#39918] [General] [answered]
hi guys.
In My page twice included JQuery
[https://easy-moving.ca](url)

`<script src="https://code.jquery.com/jquery-1.12.4.min.js"></script>`
`<script src="/media/vendor/jquery/js/jquery.min.js?3.6.0"></script>`

they comes near to each other.
How Can I disable `<script src="https://code.jquery.com/jquery-1.12.4.min.js"></script>` I guess 3.6.0 more then enough

**Comment by Fedik** [comment, marked as answer]
hi,
the version `<script src="https://code.jquery.com/jquery-1.12.4.min.js"></script>` not by Joomla core, it by one of installed extension or template on your site. You have to find which one doing it, and disable it or ask extension developer for help.

You also can try ask for assistance on joomla forum https://forum.joomla.org/

**Comment by dgrammatiko** [comment]
Probably the module mod_ebwhatsappchat is adding that. Try disabling it.

BTW for such questions (support) you should use the Forum

**Comment by Hackwar** [comment]
Since this is a support question and not a discussion, I'm closing this one.
